src/Umbraco.Tests.Integration/Umbraco.Tests.Integration.csproj

```diff
@@ -13,6 +13,16 @@
       <None Remove="TEMP\**" />
     </ItemGroup>
 
+    <ItemGroup>
+      <None Remove="Logging\logviewer.searches.config.js" />
```

Comment:
@nikolajlauridsen why do we remove this config js file and then we include it?

Reply:
My bad, probably a leftover from when I removed the file per Bjarkes review, and then added it back because it turned out that we needed it.

Reply:
Aha!

Reply:
I've removed the remove section and set the json file to content with PreserveNewest now, sorry about that.

Reply:
No worries! Otherwise good job 😉

Reply:
Thanks, hopefully there'll be a bit fewer errors next time 😀

Reply:
We all help each other to make improvements 💪